.github/workflows/go.yml

```
@@ -11,7 +11,7 @@ jobs:
   build:
     strategy:
       matrix:
-        go-version: [1.17.x, 1.18.x, 1.19.x]
+        go-version: [1.17.x, 1.18.x, 1.19.x, 1.20.x]
```

I support only the latest 3 Go versions:

```suggestion
        go-version: [1.18.x, 1.19.x, 1.20.x]
```

---

ok, cool!